### PR TITLE
readme-clas

### LINKED ISSUE #190: README for CLAS
adopt according to #184

### PR DESCRIPTION
**Pull request by albertmink**
closes #190

### CODE REVIEW

File: file-formats/clas/README.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
+`<name>.clas.abap` | 1 | [ABAP](https://help.sap.com/doc/abapdocu_cp_index_htm/CLOUD/en-US/index.htm?file=abapclass.htm) | - | [`zcl_aff_example.clas.abap`](./examples/zcl_aff_example.clas.abap)
+`<name>.clas.definition.abap` | 0...1 | | - | [`zcl_aff_example.clas.definitions.abap`](./examples/zcl_aff_example.clas.definitions.abap)
+`<name>.clas.implementations.abap` | 0...1 | | - | [`zcl_aff_example.clas.implementations.abap`](./examples/zcl_aff_example.clas.implementations.abap)
+`<name>.clas.macros.abap` | 0...1 | | - | [`zcl_aff_example.clas.macros.abap`](./examples/zcl_aff_example.clas.macros.abap)
+`<name>.clas.testclasses.abap` | 0...1 | | - | [`zcl_aff_example.clas.testclasses.abap`](./examples/zcl_aff_example.clas.testclasses.abap)
+`<name>.clas.texts.<lang>.properties` | 0...n | [Text Elements](https://help.sap.com/viewer/c238d694b825421f940829321ffa326a/7.40.19/en-US/4ec6cf916e391014adc9fffe4e204223.html) | - | [`zcl_aff_example.clas.testclasses.abap`](./examples/zcl_aff_example.clas.testclasses.abap)
```
Comment (wurzka) [inline file-formats/clas/README.md:13]: Here, the example of text elements links to the testclasses.

**Review by wurzka — CHANGES_REQUESTED** [review]
There is one (in my opinion) wrong link.

**Review by wurzka — APPROVED** [review]
Thanks, LGTM

File: file-formats/clas/README.md [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
-[`zcl_aff_example.clas.abap`](./examples/zcl_aff_example.clas.abap)                                 | Global class
-[`zcl_aff_example.clas.implementations.abap`](./examples/zcl_aff_example.clas.implementations.abap) | Local types
-[`zcl_aff_example.clas.macros.abap`](./examples/zcl_aff_example.clas.macros.abap)                   | Macros
-[`zcl_aff_example.clas.testclasses.abap`](./examples/zcl_aff_example.clas.testclasses.abap)         | Test classes
-[`zcl_aff_example.clas.texts.en.properties`](./examples/zcl_aff_example.clas.texts.en.properties)   | Translation relevant texts
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.clas.json` | 1 | [`zif_aff_clas_v1.intf.abap`](./type/zif_aff_clas_v1.intf.abap) | [`clas.json`](./clas.json) | [`zcl_aff_example.clas.json`](./examples/zcl_aff_example.clas.json)
+`<name>.clas.abap` | 1 | [ABAP](https://help.sap.com/doc/abapdocu_cp_index_htm/CLOUD/en-US/index.htm?file=abapclass.htm) | - | [`zcl_aff_example.clas.abap`](./examples/zcl_aff_example.clas.abap)
+`<name>.clas.definition.abap` | 0...1 | | - | [`zcl_aff_example.clas.definitions.abap`](./examples/zcl_aff_example.clas.definitions.abap)
```
Comment (huber-nicolas) [inline file-formats/clas/README.md:9]: <name>.clas.definition.abap
should be
<name>.clas.definition**S**.abap

Comment (albertmink) [inline file-formats/clas/README.md:9]: Implement suggestion from @huber-nicolas 
```suggestion
`<name>.clas.definitions.abap` | 0...1 | | - | [`zcl_aff_example.clas.definitions.abap`](./examples/zcl_aff_example.clas.definitions.abap)
```

**Pull request merged**